Game elements/game.js

```diff
@@ -1,10 +1,29 @@
 const mainScreen = document.querySelector('.canvas');
 const secondScreen = document.querySelector('.game__box');
 
+let closeFood = [] ;
+
+function removeFromCloseFood(food) {
```

Comment:
nie łapie tej funkcji,
Czemu robisz i++ w for, a potem i--?
Może da się to przepisac na .map?